.github/workflows/websocket-e2e-aws.yml

```diff
@@ -0,0 +1,106 @@
+name: Websocket E2E AWS
```

Comment:
Oh wait, can we just add it to create and destroy. Why create a new environment separately?

Reply:
Yeah, makes sense. Let me make the required changes and send it out for review again.